crates/gwt-core/runtime/chroma_index_runner.py

```
@@ -1673,15 +1790,263 @@ def build_embedding_document(
+    if staging is None:
+        return None
+    document_count = 0
+    try:
+        client, collection = _open_chroma_collection(
+            staging, _scope_collection_name(scope)
+        )
+        try:
+            document_count = _safe_collection_count(collection)
+        finally:
+            _close_chroma_client(client)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)